install.py

```diff
@@ -59,14 +59,11 @@ def make_package_command(package_name, package_annotations):
     Returns:
         string: A command that can be executed using the wsk cli.
     """
-    command = "{} package update {}".format(WSK_CLI, package_name) \
-        + " -a description {}".format(json.dumps(package_annotations['description'])) \
```

Comment:
What about descripton, parameteres or prettyName?

Reply:
It's only used for integration with the UI. I talked to ammar and he said since we aren't adding things to the catalog right now, we should remove it (and can add it back if thats done in the future)